15-components-lab_v16/src/00-synctext.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';

```

Comment:
Hi,
The task was to have the inputs synchronised so every time the text changes in one input the change will show on the others.

I think in your code the input fields are read only.

Am I missing something?